skimage/filters/_frangi.py

```diff
@@ -4,7 +4,7 @@
 
 
```

Comment:
I think you intended to remove `_frangi.py`? These functions are also present in `ridges.py`.